csharp-basic-topics/YieldBreakInCSharp/Tests/UtilityTests.cs

```diff
@@ -0,0 +1,27 @@
+using YieldBreakInCSharp;
+
+namespace Tests
+{
```

Comment:
Please use file-scoped namespaces. Just to avoid having one indentation lever more :)